fastlane/lanes/release.rb

```diff
@@ -213,6 +213,17 @@
     # Parse the provided version into an AppVersion object
     parsed_version = VERSION_FORMATTER.parse(new_version)
     previous_version = VERSION_FORMATTER.release_version(VERSION_CALCULATOR.previous_patch_version(version: parsed_version))
+    previous_release_branch = "release/#{previous_version}"
+
+    # Determine the base for the hotfix branch: either a tag or a release branch
+    base_ref_for_hotfix = if git_tag_exists(tag: previous_version)
```

Comment:
Same as my preliminary note on https://github.com/wordpress-mobile/WordPress-iOS/pull/24996:

```suggestion
    base_ref_for_hotfix = if git_tag_exists(tag: previous_version, remote: true)
```

Reply:
👍 Updated on f7450617d7dc29317b7c85ce42f9135979c36400.